web/pdf_page_view.js

```diff
@@ -44,41 +44,29 @@ var TEXT_LAYER_RENDER_DELAY = 200; // ms
  */
 
 /**
- * @class
  * @implements {IRenderableView}
  */
-var PDFPageView = (function PDFPageViewClosure() {
+class PDFPageView {
   /**
-   * @constructs PDFPageView
    * @param {PDFPageViewOptions} options
    */
-  function PDFPageView(options) {
-    var container = options.container;
-    var id = options.id;
-    var scale = options.scale;
-    var defaultViewport = options.defaultViewport;
```

Comment:
Similarily, let's keep `let defaultViewport = options.defaultViewport;` such that we can maintain the slightly simpler `this.pdfPageRotate = defaultViewport.rotation;` below.